Makefile

```diff
@@ -9,9 +9,15 @@ tuist/edit:
 	./make/tasks/tuist/edit.sh
 tuist/generate:
 	./make/tasks/tuist/generate.sh $(ARGS)
+tuist/build:
+	./make/tasks/tuist/build.sh $(ARGS)
```

Comment:
Just out of my curiosity.. where is $ARGS coming from?

Reply:
You can pass them when invoking make, `ARGS="..." make task`. Make doesn't support passing and reading flags :/